### PR TITLE
libical 1.0.1

### PR DESCRIPTION
**Pull request by gcoco**
Move to Version 1.0.1.
Use CMAKE build system.
Only build SHARED libs.

### CODE REVIEW

File: Library/Formula/libical.rb [hunk 1]
```diff
@@ -2,23 +2,18 @@
...
-  sha1 "25c75f6f947edb6347404a958b1444cceeb9f117"
+  url "https://github.com/libical/libical/releases/download/v1.0.1/libical-1.0.1.tar.gz"
+  sha1 "904b2c2b5c2b30f0a508f9d56eaf316dd42fc923"
 
-  bottle do
-    revision 1
-    sha1 "18d0e60043b3b78eb6872f53a0508537d7fcd5db" => :yosemite
-    sha1 "cea6dad5171431e1f7a5d7e12beb7c6eb4c3951c" => :mavericks
-    sha1 "e19ed113cb312dfeea12fba84f21f29f5866db99" => :mountain_lion
-  end
```
Comment (mikemcquaid) [inline Library/Formula/libical.rb]: Leave the bottle block as-is and it'll be updated when we pull,

File: Library/Formula/libical.rb [hunk 2]
```diff
@@ -2,23 +2,18 @@
...
+  depends_on "cmake" => :build
 
   def install
-    system "./bootstrap"
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    cmake_args = std_cmake_args
+    cmake_args << "-DSHARED_ONLY=true"
+    cmake_args << ".."
+    mkdir 'build' do
```
Comment (mikemcquaid) [inline Library/Formula/libical.rb]: Use double-quotes everywhere.

File: Library/Formula/libical.rb [hunk 3]
```diff
@@ -2,23 +2,18 @@
...
+      system "cmake", *cmake_args
+      system "make install"
```
Comment (mikemcquaid) [inline Library/Formula/libical.rb]: `system "make", "install"`

File: Library/Formula/libical.rb [hunk 4]
```diff
@@ -2,23 +2,18 @@
...
 
   def install
-    system "./bootstrap"
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    cmake_args = std_cmake_args
+    cmake_args << "-DSHARED_ONLY=true"
+    cmake_args << ".."
+    mkdir 'build' do
+      system "cmake", *cmake_args
```
Comment (mikemcquaid) [inline Library/Formula/libical.rb]: `system "cmake", "..", "-DSHARED_ONLY=true", *std_cmake_args`

**Comment by gcoco** [comment]
Will resubmit request.

**Comment by tdsmith** [comment]
That's fine; you can also continue pushing commits to this branch (the one you opened a pull request against).

**Pull request closed without merging**